### PR TITLE
[13.0][MIG]partner_fax

### PR DESCRIPTION
**Pull request by ps-tubtim**
This module adds a fax field into the partner form.

### CODE REVIEW

File: partner_fax/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    'name': 'Partner fax',
+    'category': 'Extra Tools',
+    'summary': 'Add fax number on partner',
+    'version': '13.0.1.0.0',
+    'license': 'AGPL-3',
+    'author':  'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/partner-contact',
+    'depends': [
+        'base_setup'
+        ],
```
Comment (NL66278) [inline partner_fax/__manifest__.py]: wrong indent (and also below on data tag)

File: partner_fax/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (NL66278) [inline partner_fax/__init__.py]: Make licence link https: instead of http: here and in other files.

**Review by NL66278 — CHANGES_REQUESTED** [review]
Small things, but easy to correct. See other comments.

**Comment by ps-tubtim** [comment]
@NL66278 Thank you for suggestions.

**Review by NL66278 — APPROVED** [review]
:+1:

**Review by astirpe — APPROVED** [review]
Tested in Runbot: OK. Code review: OK.

**Pull request merged**